charts/sourcegraph/templates/jaeger/jaeger-collector.Service.yaml

```diff
@@ -30,8 +30,9 @@ spec:
     protocol: TCP
     targetPort: 14250
   selector:
-    {{- include "sourcegraph.selectorLabels" . | nindent 4 }}
-    app.kubernetes.io/name: jaeger
+    {{- include "sourcegraph.jaeger.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
It looks like the `app.kubernetes.io/instance` labeled isn't being attached to the pods - I think this selector and the service selectors will fail?

Reply:
the diff is outdated and I just updated the comment https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/62#issuecomment-1063412878

https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/62/files#diff-3911ffe87f6289604d636cc83ba2a3df8dab9685b2c07c4e49763eb85a87897bR44